closure/templates/test/BUILD

```diff
@@ -116,6 +128,18 @@ closure_js_test(
     ],
 )
 
+closure_js_binary(
+    name = "greeter_idom_fail_bin",
+    language = "ECMASCRIPT5_STRICT",
+    deps = [
+        ":greeter_idom_fail_lib",
+        "//closure/library",
+        "//closure/library:testing",
+    ],
+    pedantic = True,
+    internal_expect_failure = True,
```

Comment:
I absolutely love the fact that you're testing failure conditions. I had to put a lot of work into making that possible.